hack/generate-docs.go

```diff
@@ -29,9 +29,9 @@ func main() {
 	if len(os.Args) > 1 {
 		dir = os.Args[1]
 	}
-	err := doc.GenMarkdownTree(rootCmd, dir + "/docs/cmd/")
+	err := doc.GenMarkdownTree(rootCmd, dir+"/docs/cmd/")
```

Comment:
would revert that as it's not related to this PR.

Reply:
When i ran go fmt for complete client code its modified  i have reverted back now.